examples/tracing-http-propagator/Cargo.toml

```diff
@@ -23,5 +23,8 @@ tokio = { workspace = true, features = ["full"] }
 opentelemetry = { path = "../../opentelemetry" }
 opentelemetry_sdk = { path = "../../opentelemetry-sdk" }
 opentelemetry-http = { path = "../../opentelemetry-http" }
-opentelemetry-stdout = { workspace = true, features = ["trace"] }
+opentelemetry-stdout = { workspace = true, features = ["trace", "logs"] }
```

Comment:
This isn't related to this PR, but I feel a bit funny about using workspace deps _in the examples_, as it prevents the user from simply copying the example and using it.

Reply:
once we move docs to opentelemetry.io, this would be proper examples that can be just copy-pasted.